### PR TITLE
DBZ-461 Changed default bigint conversion to long

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-461

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java [hunk 1]
```diff
@@ -718,7 +718,7 @@ public static EventDeserializationFailureHandlingMode parse(String value) {
 
     public static final Field BIGINT_UNSIGNED_HANDLING_MODE = Field.create("bigint.unsigned.handling.mode")
                                                            .withDisplayName("BIGINT UNSIGNED Handling")
-                                                           .withEnum(BigIntUnsignedHandlingMode.class, BigIntUnsignedHandlingMode.PRECISE)
+                                                           .withEnum(BigIntUnsignedHandlingMode.class, BigIntUnsignedHandlingMode.LONG)
                                                            .withWidth(Width.SHORT)
                                                            .withImportance(Importance.MEDIUM)
                                                            .withDescription("Specify how BIGINT UNSIGNED columns should be represented in change events, including:"
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: The description needs updating, too.

Comment (jpechane) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: Fixed

**Comment by jpechane** [comment]
The docs PR has been created https://github.com/debezium/debezium.github.io/pull/90

**Comment by gunnarmorling** [comment]
Thanks, @jpechane!

**Pull request merged**